moment.parseFormat.js

```diff
@@ -95,17 +100,17 @@
     format = format.replace(regexTimezone, 'Z');
 
     // 05:30:20pm ☛ hh:mm:ssa
-    format = format.replace(regexHoursWithLeadingZeroDigitMinutesSecondsAmPm, 'hh:mm:ss$1a');
+    format = format.replace(regexHoursWithLeadingZeroDigitMinutesSecondsAmPm, appendAMPMCase(format, 'hh:mm:ss$1'));
```

Comment:
Actually, can you elaborate more on that solution?  We still have to append the correct 'a' or 'A' to the format based on the AM/pm casing.  Right now its hardcoded to lowercase 'a'.  Another way we can avoid 6 calls, is by adding the 'A' character around line 114 when the time formats have run.  Thoughts?